content/news/XXXX-XX-XX-mixxx-2-3-1-released.md

```diff
@@ -0,0 +1,19 @@
+title: Mixxx 2.3.1 released
+authors: Daniel Schürmann
+tags: 2.3, 2.3.1, release announcement
+status: draft
```

Comment:
I guess the draft status should also be removed before merging?

Reply:
No, from the README:
> Let the status on draft and don't add a date field. When the PR is merged, GitHub actions will automatically rename the file, add the appropriate date and remove the draft status.